### PR TITLE
Add additional information and blueprint

### PR DESCRIPTION
**Pull request by emilhorlyck**
Relating to issue #16712 https://github.com/home-assistant/home-assistant.io/issues/16712

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

This is an addition of information to give context to new users. Furthermore, a blueprint is added to get users started.

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: #16712

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @eHorlyck, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17342).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/getting-started/presence-detection.markdown [hunk 1]
```diff
@@ -15,12 +15,16 @@ Screenshot of Home Assistant showing a school, work and home zone and two people
...
 
 During the setup of Home Assistant Companion on your mobile device, the app will ask for permission to allow the device's location to be provided to Home Assistant. Allowing this will create a `device_tracker` entity for that device which can be used in automations and conditions.
 
+### Room based presence detection
+
+Presence detetion is not limited to tell if a you are home or not. You can use PIR motion sensors which is provided by brands several brands. The sensor will be exposed as a [binary sensor][binary-sensor] inside Home Assistant and can then be used to trigger all kinds of cool automations. The most basic is to control the lights. This can be done using this blueprint.
+[![Open your Home Assistant instance and show the blueprint import dialog with a specific blueprint pre-filled.](https://my.home-assistant.io/badges/blueprint_import.svg)](https://my.home-assistant.io/redirect/blueprint_import/?blueprint_url=https%3A%2F%2Fmy.home-assistant.io%2Fredirect%2Fblueprint_import%2F%3Fblueprint_url%3Dhttps%253A%252F%252Fcommunity.home-assistant.io%252Ft%252Fturn-light-on-and-off-based-on-detected-motion%252F255994)
```
Comment (frenck) [inline source/getting-started/presence-detection.markdown]: This is a third-party Blueprint, and should not be included directly in the documentation like this. We currently do not have a method for correctly including and storing Blueprints in our website.

File: source/getting-started/presence-detection.markdown [hunk 2]
```diff
@@ -15,12 +15,16 @@ Screenshot of Home Assistant showing a school, work and home zone and two people
 
 ### Adding presence detection
 
-There are different ways of setting up presence detection. Usually the easiest way to detect presence is by checking which devices are connected to the network. You can do that if you have one of our [supported routers][routers]. By leveraging what your router already knows, you can easily detect if people are at home.
+There are different ways of setting up presence detection using the official intergrations. Usually the easiest way to detect presence is by checking which devices are connected to the network. You can do that if you have one of our [supported routers][routers] which will provide information on which devices are connected. By leveraging what your router already knows, you can easily detect if people are at home. One thing to consider is that some devices drop of the network to save battery, which could lead to false state changes.
```
Comment (frenck) [inline source/getting-started/presence-detection.markdown:18]: - `using the official integrations` -> What does that mean? Why is that added?
- `One thing to consider is that some devices drop of the network to save battery, which could lead to false state changes.` -> Not sure if a new users understands this line.

Considering the goal here, I'm not sure if we should talk about routers at all. Those devices tend to spawn a lot of entities that only add more confusion.

File: source/getting-started/presence-detection.markdown [hunk 3]
```diff
@@ -32,6 +36,7 @@ Zones allow you to name areas on a map. These areas can then be used to name the
 The map view will hide all devices that are home.
 </div>
 
+[binary-sensor] https://www.home-assistant.io/integrations/binary_sensor/
```
Comment (frenck) [inline source/getting-started/presence-detection.markdown:38]: - This is broken Markdown.
- Don't use full links.

File: source/getting-started/presence-detection.markdown [hunk 4]
```diff
@@ -15,12 +15,16 @@ Screenshot of Home Assistant showing a school, work and home zone and two people
...
+It's also possible to run an app on your phone to provide detailed location information to your Home Assistant instance. For iOS and Android, we suggest using the [Home Assistant Companion app][companion]. The easiest way to configure remote access is to subscribe to [Nabu casa][nabu-casa]. This service removes the need to handle router settings, SSL certificates, and so much more.
 
 During the setup of Home Assistant Companion on your mobile device, the app will ask for permission to allow the device's location to be provided to Home Assistant. Allowing this will create a `device_tracker` entity for that device which can be used in automations and conditions.
 
+### Room based presence detection
+
+Presence detetion is not limited to tell if a you are home or not. You can use PIR motion sensors which is provided by brands several brands. The sensor will be exposed as a [binary sensor][binary-sensor] inside Home Assistant and can then be used to trigger all kinds of cool automations. The most basic is to control the lights. This can be done using this blueprint.
```
Comment (frenck) [inline source/getting-started/presence-detection.markdown]: - `detetion` -> `detection`
- `if a you are`
- `which is provided` -> `which are provided`

At this point in the guide, there have been 0 references to Blueprints. To the end reader, this doesn't mean anything yet.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/getting-started/presence-detection.markdown (hunk 1 above)
Comment (eHorlyck) [reply source/getting-started/presence-detection.markdown]: Removed blueprint

**Comment by frenck** [comment]
@eHorlyck Why did you close this?

**Comment by emilhorlyck** [comment]
Hey @frenck 

I'm honestly challenged by how I practically submit changes in the pull request. Making a pull request and using GitHub to collaborate is still very new to me. So I closed it to clean up. 

Please correct me if that isn't the procedure.

**Pull request closed without merging**